app/src/main/java/com/example/househomey/Filters/MakeFilterFragment.java

```diff
@@ -1,15 +1,31 @@
 package com.example.househomey.Filters;
 
 import android.view.View;
+import android.widget.EditText;
 
-public class MakeFilterFragment extends FilterFragment {
+import com.example.househomey.R;
```

Comment:
What does this import do?

Reply:
Yeah I believe it's because we moved the Filters into their own package. R exists in the main package `com.example.househomey`, but not `com.example.househomey.filters` so we need to import it

Reply:
Ahhh, I could not find the line using it, makes sense.